turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -498,6 +501,13 @@ impl<B: BackingStorage> TurboTasksBackendInner<B> {
                     let _ = reader_task.add(CachedDataItem::CellDependency { target, value: () });
                 }
             }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +484 to +503]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Unrelated question: How expensive do you think this is? How often do you think we read the same cell multiple times in the same execution?

Would it make sense to keep a task-local set of cells we've already marked ourselves dependent on during this task execution, and skip doing this work if that set contains this cell?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/vercel/next.js/72330?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->